src/appleseed/renderer/modeling/bsdf/diffusebtdf.cpp

```diff
@@ -139,7 +139,7 @@ namespace
                 sample.m_value.m_diffuse = values->m_transmittance;
                 sample.m_value.m_diffuse *= values->m_transmittance_multiplier * RcpPi<float>();
                 sample.m_value.m_beauty = sample.m_value.m_diffuse;
-                sample.m_aov_components.m_albedo = values->m_transmittance;
+                sample.m_aov_components.m_albedo = sample.m_value.m_diffuse;
```

Comment:
What we really want is values->m_transmittance * values->m_transmittance_multiplier, 
without the RcpPi<float>() factor.

What you can do is compute the albedo first and then:

diffuse = albedo;
diffuse *= RcpPi<float>();

Reply:
I submitted a new change, but we want albedo = diffuse right?